settingshelper.py

```diff
@@ -199,3 +200,37 @@ def fix_logger_obfuscation(fix_logger_obfuscation_, logging_config):
                         'logging.StreamHandler'
                     ), file=sys.stderr)
                 handler["class"] = "logging.StreamHandler"
+
+
+def configure_sentry(base_dir, server_env, pub_key, priv_key, project_id):
+    if not (pub_key and priv_key and project_id):
+        return
+
+    try:
+        from raven import breadcrumbs, fetch_git_sha
+    except ImportError:
+        return
+
+    release_dir = base_dir.split('/')[-1]
+    if re.match('\d{4}-\d{2}-\d{2}_\d{2}.\d{2}', release_dir):
+        release = "{}_{}".format(server_env, release_dir)
```

Comment:
(i'm also happy to merge this now if you'd like and you can tinker with it later)